### DISCUSSION TITLE
Is there frambuffer or queue can support  tuple

### DISCUSSION THREAD

**Discussion by skylin008** [micropython/micropython#13267] [STM32 / Pyboard]
Merry Christmas ! Is there frambuffer or queue can support  tuple , thanks!

**Comment by peterhinch** [comment]
The `asyncio` [Queue class](https://github.com/peterhinch/micropython-async/blob/master/v3/docs/TUTORIAL.md#35-queue) allows any object to be queued, including tuples.
